modules/speech-to-text-impl/src/main/java/org/opencastproject/speechtotext/impl/engine/VoskEngine.java

```diff
@@ -120,8 +139,11 @@ public File generateSubtitlesFile(File mediaFile, File preparedOutputFile, Strin
     } finally {
       IoSupport.closeQuietly(process);
     }
+    List<Object> returnValues = new ArrayList<>();
```

Comment:
Why not a `Map<String, File>` where the key is the lang?

Reply:
That's … a good idea! Easier to read.

Reply:
Done!